### PR TITLE
Persist Twitch VOD URLs for replays to DynamoDB

### PR DESCRIPTION
**Pull request by joolean**

### CODE REVIEW

File: hsreplaynet/games/processing.py [hunk 1]
```diff
@@ -983,6 +986,100 @@ def update_game_meta(parser, meta):
 		meta["game_type"] = int(parser.game_meta["GameType"].as_bnet(wild=is_wild))
 
 
+TWITCH_VOD_URL_PATTERN = re.compile(r"https://www\.twitch\.tv/twitch/v/\d+")
```
Comment (beheh) [inline hsreplaynet/games/processing.py]: We should prefix with `^`.

File: hsreplaynet/admin/management/commands/create_dynamodb_tables.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import importlib
+
+from django.core.management import BaseCommand
+
+from hsreplaynet.local_settings import DYNAMODB_TABLES
```
Comment (beheh) [inline hsreplaynet/admin/management/commands/create_dynamodb_tables.py]: You should import settings using `from django.conf import settings`, as that resolves `settings`, `local_settings` and the test `settings`. That should hopefully fix pytest on Travis CI.

File: hsreplaynet/games/processing.py [hunk 3]
```diff
@@ -983,6 +986,100 @@ def update_game_meta(parser, meta):
...
+		)
+
+		if friendly_deck.archetype:
+			twitch_vod.friendly_player_archetype_id = friendly_deck.archetype.id
+
+		# Generate the "combined rank" synthetic range key.
+
+		if friendly_player.rank == 0:
+			twitch_vod.legend_rank = friendly_player.legend_rank
+			twitch_vod.combined_rank = "L%s:R%s" % (
```
Comment (azeier) [inline hsreplaynet/games/processing.py]: What exactly is this and how is it used?

Comment (azeier) [reply hsreplaynet/games/processing.py]: it's always going to be `L:{legend_rank}:R0`

**Pull request merged**